command/identify.py

```diff
@@ -1,13 +1,16 @@
 import yaml
 
 def get_operator_artifact_type(operatorArtifactString):
+    """get_operator_artifact_type takes a yaml string and determines if it is one of the 
+    expected bundle types: ClusterServiceVersion, CustomResourceDefinition, or Package.
+
+    :param operatorArtifactString: Yaml string to type check
+    """
     operatorArtifact = yaml.load(operatorArtifactString)
     if isinstance(operatorArtifact, dict):
         if "packageName" in operatorArtifact:
-            return "packages"
+            return "Package"
         elif "kind" in operatorArtifact:
-            if operatorArtifact["kind"] == "ClusterServiceVersion": 
-                return "clusterServiceVersions"
-            elif operatorArtifact["kind"] == "CustomResourceDefinition":
-                return "customResourceDefinitions"
-    return "invalid"
+            if operatorArtifact["kind"] == "ClusterServiceVersion" or operatorArtifact["kind"] == "CustomResourceDefinition":
```

Comment:
just protip `if operatorArtifact["kind"] in ("ClusterServiceVersion", "CustomResourceDefinition"):`

Reply:
@MartinBasti addressed in latest commit.